tests/foreman/ui/test_org.py

```diff
@@ -584,22 +591,24 @@ def test_add_hostgroup(self, host_grp_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        host_grp = entities.HostGroup(name=host_grp_name).create()
-        self.assertEqual(host_grp.name, host_grp_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_hostgroups=[host_grp_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_hostgrps'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % host_grp_name))
-            self.assertIsNotNone(element)
+            for host_grp_name in generate_strings_list():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.